ws-nextjs-app/package.json

```diff
@@ -28,9 +28,9 @@
     "@emotion/babel-plugin": "11.11.0",
     "@emotion/jest": "^11.11.0",
     "@testing-library/jest-dom": "6.4.2",
-    "@testing-library/react": "15.0.2",
+    "@testing-library/react": "15.0.4",
     "babel-plugin-transform-rename-import": "2.3.0",
-    "cypress": "13.8.0",
+    "cypress": "13.8.1",
```

Comment:
@amoore108 does ws-nextjs-app inherit packages from the main package.json file? And if so, should we consider removing the dupes from here?